Src/IronPython/Runtime/Operations/StringOps.cs

```diff
@@ -23,6 +25,10 @@
 
 using SpecialNameAttribute = System.Runtime.CompilerServices.SpecialNameAttribute;
 using DisallowNullAttribute = System.Diagnostics.CodeAnalysis.DisallowNullAttribute;
+using NotNullWhenAttribute = System.Diagnostics.CodeAnalysis.NotNullWhenAttribute;
```

Comment:
Will see how it works.

`IronPython.Hosting`? Where did it come from? I didn't add it and see no reason why it might be needed. I did use the VS refactoring function a couple of times (symbol rename), it must have added it then automatically, don't know why.